setup.py

```diff
@@ -1,2 +1,2 @@
 import os
-os.system("python3 ./utilities/svg_parsing/svg_parser.py > ./hs/SVGGen.hs")
+os.system("python ./utilities/svg_parsing/svg_parser.py > ./hs/SVGGen.hs")
```

Comment:
@foreverwander I think this setup.py file is problematic for this reason, but you shouldn't change it in this pull request. (So you should add another commit to remove it.)